gcc/testsuite/rust/compile/inline_1.rs

```diff
@@ -0,0 +1,15 @@
+#[inline]
+fn test_a() {}
+
+// { dg-final { scan-tree-dump-times {always_inline} 1 gimple } }
```

Comment:
This produces:

```
PASS: rust/compile/inline_1.rs (test for excess errors)
UNRESOLVED: rust/compile/inline_1.rs scan-tree-dump-times gimple "always_inline" 1
```

```
                === rust Summary for unix ===

# of expected passes            5746
# of expected failures          21
# of unresolved testcases       1
```

1. Our automated testing shouldn't have let that pass.
2. Add the missing `-fdump-tree-gimple`.